package.json

```diff
@@ -120,10 +120,14 @@
     "concurrently": "^6.2.0",
     "copyfiles": "^2.1.0",
     "lerna": "^4.0.0",
+    "npm": "^6.14.18",
```

Comment:
I don't think we want to take a dependency on npm. Just do a global install of it.